tests/UI/campaigns/functional/FO/03_userAccount/01_orderHistory/reorderFromOrderList.js

```diff
@@ -0,0 +1,244 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+
+// Import pages
+// FO pages
+const foHomePage = require('@pages/FO/home');
+const foLoginPage = require('@pages/FO/login');
+const foMyAccountPage = require('@pages/FO/myAccount');
+const cartPage = require('@pages/FO/cart');
+const checkoutPage = require('@pages/FO/checkout');
+const orderConfirmationPage = require('@pages/FO/checkout/orderConfirmation');
+const foOrderHistoryPage = require('@pages/FO/myAccount/orderHistory');
+const orderDetailsPage = require('@pages/FO/myAccount/orderDetails');
+
+// Import datas
+const {DefaultCustomer} = require('@data/demo/customer');
+const CartData = require('@data/FO/cart');
+const {PaymentMethods} = require('@data/demo/paymentMethods');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+// context
+const baseContext = 'functional_FO_userAccount_orderHistory_reorderFromOrderList';
```

Comment:
Shouldn't be `functional_FO_userAccount_orderHistory_orderDetails_reorderFromOrderList` ?

Reply:
:ok: